components/MembersPics/MembersPics.tsx

```diff
@@ -25,7 +26,7 @@ const CommunityLink = () => {
   return (
     <p className="text-3xl md:text-[60px] font-bold pr-5">
       הכירו את חברי{' '}
-      <Link className="text-purple-500 hover:underline" href="/members">
+      <Link className="text-purple-500 hover:underline" href={LINKS.MEMBERS}>
         הקהילה
```

Comment:
Opinion - I would change this into `COMMUNITY_MEMBERS` because `MEMERS` feels just too generic. 
A new contributor to the project might think that this is related to the site members (LIke the you sign via authentication)  and not the community members. 

Then again this is jys opinion, what you did is ok as well.

Reply:
Good call.
Changes applied.

Reply:
Good call.
Changes applied.

Reply:
Ill leave that to another PR.
The benefit would be minor tbh